### PR TITLE
Added FAQs

### LINKED ISSUE #9: FAQ Page
## Purpose
An easy place to share answers to the commonly asked questions - things like "How long is x path?", "What course should I start with?"

### Is your feature request related to a problem? Please describe.
There tends to be a lot of questions posted to the server that are commonly asked, and it would cut down on the number of repeated questions to have them answered in one place.

## Proposed Solution
Anything that works!

## Acceptance Criteria
Please list the requirements the implementation of this feature should meet.
- [ ] Page should have some of the most commonly asked questions and their answers (potentially with links to further resources)
- [ ] If you can wireframe, a screenshot of that would help!

### PR DESCRIPTION
**Pull request by lyallstewart**
## What issue is this solving?

Closes #9 

### Description
Adds a responsive FAQ page with a couple of the questions that I've seen most commonly around the server and adds the page to the homepage and the sites navbars. The actual list of the FAQs will probably need to be added to as we go but I figured it would be better to get it set up with a handful first.
image

## Any helpful knowledge/context for the reviewer?

- Any new dependencies to install? ❌ 
- Any special requirements to test? ❌ 

### Feelings gif (optional)
Dog with hotdog gif

### Please make sure you've attempted to meet the following coding standards

- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: faqs.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+        </nav>
+    </header>
+    <main class="has-text-centered">
+        <h1 class="is-size-1">Frequently Asked Questions</h1>
+        <section class="container has-background-white p-4">
+            <h2 class="is-size-2">Q: Is Codecademy Down?</h2>
+            <div class="content">
+                <p>
+                    <b>A:</b> Most issues connecting to codecademy are caused by a local issue, if you are struggling to
```
Comment (aedwardg) [inline faqs.html:44]:
```suggestion
                    <b>A:</b> Most issues connecting to codecademy are caused by a local issue. If you are struggling to
```

File: faqs.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                <a href="https://help.codecademy.com/hc/en-us/requests/new">Submit a Request</a>
+            </div>
+
+            <hr>
+
+            <h2>Q: How long does [x] course take?</h2>
+            <div class="content">
+                <p>
+                    <b>A:</b> Estimates for the lengths of all Codecademy courses should be displayed in the catalogue page
+                    for the relevant course however it is important to remember that these are just estimates, and the actual
```
Comment (aedwardg) [inline faqs.html:59]:
```suggestion
                    for the relevant course. However, it is important to remember that these are just estimates, and the actual
```

File: faqs.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                <a href="https://help.codecademy.com/hc/en-us/articles/360012296653-Estimated-Career-Path-Completion-Times">Career Paths</a><br>
+                <a href="https://help.codecademy.com/hc/en-us/articles/360022742233-Estimated-Skill-Path-Completion-Times">Skill Paths</a>
+            </div>
+
+            <hr>
+
+            <h2 class="is-size-2">Q: I have an issue with my account (including billing issues such as refunds)</h2>
+            <div class="content">
+                <p>
+                    <b>A:</b> As we are a primarily community-run server we do not have the ability to to assist with billing issues,
```
Comment (aedwardg) [inline faqs.html:72]:
```suggestion
                    <b>A:</b> As we are a primarily community-run server we do not have the ability to assist with billing issues,
```

**Review by aedwardg — APPROVED** [review]
Looks good, just a few grammatical errors that need to be fixed and then it should be good to go 👍

**Comment by lyallstewart** [comment]
Thanks @aedwardg!

**Pull request merged**